web/themes/new_weather_theme/templates/partials/radar.html.twig

```diff
@@ -2,14 +2,17 @@
 
 <div class="grid-container margin-top-2 padding-x-0 tablet:padding-x-2" wx-outer-radar-container>
   <div class="grid-row tablet-lg:grid-col-8 tablet-lg:grid-offset-2">
-    <h3 class="visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">Radar</h3>
+    <h3 class="visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">{{ "Radar" | t }}</h3>
   </div>
 
   <div class="grid-container padding-0 bg-white shadow-1 grid-col-12 tablet-lg:grid-offset-2 tablet-lg:grid-col-8">
     <div class="usa-sr-only">
-      <p>Listed below is a visual representation of radar data for
-      {{ weather.place }}. For more details on specific weather conditions,
-      view hourly data within the <a href="" data-daily-tab-link>daily tab.</a></p>
+      <p>
```

Comment:
Thanks for thinking to include this. Rather than the paragraph here I am thinking this should be be a (sr-only) `caption` within the table itself

Reply:
oh  misinterpreted those one. You can ignore